requirements.txt

```diff
@@ -1,3 +1,4 @@
+bluepy==1.0.5
```

Comment:
We didn't discuss this yesterday.  It should work fine on Ubuntu, Rasbian OSes.  We can't install libglib2.0 in Snappy Ubuntu Core.

Regarding import error, I missed adding bluepy in requirements.py of edge_intelligence branch.